kong/pdk/node.lua

```diff
@@ -255,6 +257,22 @@ local function new(self)
     return gsub(hostname, "\n$", "")
   end
 
+
+  local prefix = self and self.configuration and self.configuration.prefix
+  if prefix and self.configuration.role == "data_plane" then
+    local id, err = private_node.load_node_id(prefix)
+    if id then
+      node_id = id
+      ngx.log(ngx.INFO, "restored node_id from the filesystem: ", node_id)
+
+    else
+      id = _NODE.get_id()
+      if err then
+        ngx.log(ngx.WARN, "failed to restored node_id from the filesystem: " .. err, " (generated a new one " .. id .. ")")
```

Comment:
removed string concat and fixed a typo

```suggestion
        ngx.log(ngx.WARN, "failed to restore node_id from the filesystem: ", err, " (generated a new one: ", id, ")")
```